primitives/cnight-observation/src/lib.rs

```diff
@@ -78,25 +78,27 @@ impl TryFrom<Vec<u8>> for CardanoRewardAddressBytes {
 	PartialOrd,
 	Ord,
 )]
-pub struct DustPublicKeyBytes(#[serde(with = "hex")] pub [u8; 33]);
+pub struct DustPublicKeyBytes(pub BoundedVec<u8, ConstU32<33>>);
```

Comment:
The length is variable according to Thomas and could change later.

Reply:
The underlying representation is has a 33 byte max length with the current ledger version - if this changes, we can update this bound